.github/ISSUE_TEMPLATE/build-a-component.md

```diff
@@ -0,0 +1,57 @@
+---
+name: "Build a component"
+about: Everything we need to do when we build a component
+title: ''
+labels: "contribution or major iteration, design, epic"
+assignees: ''
+
+---
+
+## What
+<!-- Add name of component, plus any relevant links (for example, community backlog issues and comments, pull requests for the component and its guidance) -->
+
+## Why
+<!-- Add reason for building the component -->
+
+## Who needs to know about this
+<!-- Add team roles involved in building the component -->
+
+## Acceptance criteria
+<!-- Customise, and add component-specific checklist items to, this list of criteria all components need to meet -->
+### Accessibility
+In general, the component must:
+- [ ] work with [assistive technologies listed in the service manual(https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices)
+- [ ] pass tests on tools like axe and HTML validator
+- [ ] works when zoomed at high resolution
+- [ ] work in high contrast mode
+- [ ] follow [focus state guidance](https://design-system.service.gov.uk/get-started/focus-states/)
+
+For more information on accessibility testing, [read our documentation on testing components](https://github.com/alphagov/govuk-frontend/blob/main/docs/contributing/test-components-using-accessibility-acceptance-criteria.md).
+
+### Coding standards
+The component's guidance must include these sections:
+- [ ] have Nunjucks options that are consistent with other Design System components
```

Comment:
Do we want to link to https://github.com/alphagov/govuk-frontend/blob/main/docs/contributing/coding-standards/nunjucks-api.md here?